widget/embedded/src/store/notification.ts

```diff
@@ -54,23 +61,61 @@ export const useNotificationStore = createSelectors(
           }));
         },
         setAsRead: (requestId) => {
-          set((state) => ({
-            notifications: state.notifications.map((notificationItem) =>
-              notificationItem.requestId === requestId
-                ? { ...notificationItem, read: true }
-                : notificationItem
-            ),
-          }));
+          set((state) => {
+            const notifications: Notification[] = [];
```

Comment:
This function can be very simple to clear only the notification.
(remove `isSwapFinished` from this logic)